### ISSUE TITLE
Compass, user dot, annotations are inaccessible on iOS

### ISSUE THREAD

**Issue by 1ec5** [mapbox/mapbox-gl-native#1493]
With VoiceOver enabled, the attribution button and annotation callouts can receive focus, but the compass, user dot, and annotation pins cannot. The pins are rendered directly in GL, but there should still be a way to give them accessibility regions. The other views all have accessibility labels.

**Comment by incanus** [comment]
> there should still be a way to give them accessibility regions

Oof, that might be tricky, depending on how we do it. Realtime means we'd have to mirror annotations to the view hierachy, which is going to be high overhead. Doing so after a small delay might be a decent approach.

**Comment by 1ec5** [comment]
I’m looking at creating `UIAccessibilityElements` on the fly as UIAccessibility asks for them. These elements don’t have to be part of the view hierarchy, so there’s a lot less overhead.

**Comment by incanus** [comment]
Ah, nice!
